app/controllers/editorial/news_controller.rb

```diff
@@ -0,0 +1,48 @@
+module Editorial
```

Comment:
I'm a bit confused as to why this controller is needed? Why not just editorial/nodes ?